src/lib/DefaultRuleManager.ts

```diff
@@ -308,9 +309,9 @@ export class DefaultRuleManager implements RuleManager {
 					// If the target is a directory, we should get everything in it, convert relative paths to absolute
 					// paths, and then filter based our matcher.
 					const relativePaths = await globby(targetPath);
-					const ruleTarget = {
+					const ruleTarget: RuleTarget = {
 						target: targetPath,
-						isDirectory: true,
+						targetType: TargetType.DIRECTORY,
```

Comment:
Correct. The enum allows us to indicate exactly which type of target a given target is, instead of just whether it's a directory like we were previously doing.